### PR TITLE
New VSHmenu

### PR DESCRIPTION
**Pull request by pyroesp**
Improved the way the VSHMenu is displayed.
Main menu:
- fully center-justified

Advanced menu:
- menu strings with options are left-justified with their options next to it
- menu strings without are center-justified

No more need for whitespaces in the satelite files to align everything properly.
This is now done automatically.

### CODE REVIEW

File: extras/menus/advancedvsh/advanced.c [hunk 1]
```diff
@@ -46,385 +46,416 @@ extern char* available_fonts[];
...
+				break;
+			default:	
+				bc = (submax_menu==submenu_sel) ? 0xff8080 : 0x0000a5ff;
+				blit_set_color(fc,bc);
+		}
 
+		// do fg_color stuff
+		switch(config.vsh_fg_color) {
+			// Random
+			case 0:
```
Comment (krazynez) [inline extras/menus/advancedvsh/advanced.c:242]: Looks like this was an issue before, but 0 would fall through to then be case 1, should have a break possibly.

File: extras/menus/injector/Makefile
Comment (krazynez) [inline extras/menus/injector/Makefile:1]: We don't need this really.

File: extras/menus/injector/exports.exp
Comment (krazynez) [inline extras/menus/injector/exports.exp:1]: or this

File: extras/menus/injector/include.bak/module2.h
Comment (krazynez) [inline extras/menus/injector/include.bak/module2.h:1]: or this

File: extras/menus/injector/include.bak/systemctrl.h
Comment (krazynez) [inline extras/menus/injector/include.bak/systemctrl.h:1]: " "

File: extras/modules/post_injector/Makefile
Comment (krazynez) [inline extras/modules/post_injector/Makefile:1]: Not needed

File: extras/modules/post_injector/exports.exp
Comment (krazynez) [inline extras/modules/post_injector/exports.exp:1]: " "

File: extras/modules/post_injector/main.c
Comment (krazynez) [inline extras/modules/post_injector/main.c:1]: " '

File: loader/live/user/hello_world/main.s
Comment (krazynez) [inline loader/live/user/hello_world/main.s:1]: Why is this here?

**Review by krazynez — CHANGES_REQUESTED** [review]
To many things extra added. Please just add advancedvsh, fonts, and translations.

**Comment by pyroesp** [comment]
> To many things extra added. Please just add advancedvsh, fonts, and translations.

@krazynez All things indicated are from the first rebase commit on this branch, which I mentioned on discord.
No worries I'll just delete this branch, then create a new branch from main and push there.

**Pull request closed without merging**